packages/cardano-graphql-services/src/StakePoolSearch/StakePoolSearchHttpServer.ts

```diff
@@ -0,0 +1,71 @@
+import { Cardano } from '@cardano-sdk/core';
+import { DbSyncStakePoolSearchProvider } from './DbSyncStakePoolSearchProvider';
+import { ErrorObject, serializeError } from 'serialize-error';
+import { HttpServer } from '../Http';
+import { Logger, dummyLogger } from 'ts-log';
+import bodyParser, { Options } from 'body-parser';
+import express, { Router } from 'express';
+import net from 'net';
+
+export interface StakePoolSearchServerDependencies {
+  stakePoolSearchProvider: DbSyncStakePoolSearchProvider;
+  logger?: Logger;
+}
+
+export interface StakePoolSearchServerConfig {
+  listen: net.ListenOptions;
+  bodyParser?: {
+    limit?: Options['limit'];
+  };
+}
+
+export interface StakePoolSearchResponse {
+  stakePools: Cardano.StakePool[];
+}
+
+export class StakePoolSearchHttpServer extends HttpServer {
+  private constructor(
+    config: StakePoolSearchServerConfig,
+    router: Router,
+    dependencies: StakePoolSearchServerDependencies
+  ) {
+    super({ listen: config.listen, name: 'StakePoolSearchServer', router }, dependencies.logger);
+  }
+  static create(
+    config: StakePoolSearchServerConfig,
+    { stakePoolSearchProvider, logger = dummyLogger }: StakePoolSearchServerDependencies
+  ) {
+    const router = express.Router();
+    router.use(bodyParser.raw({ limit: config.bodyParser?.limit || '500kB', type: 'application/json' }));
+    router.get('/health', async (req, res) => {
```

Comment:
This is being lifted to the base class in https://github.com/input-output-hk/cardano-js-sdk/pull/213. When rebasing you'll just need to pass in a callback function for the health check

Reply:
Will work in the next conmmit on http server :+1: